components/table/table-col-sort-button.js

```diff
@@ -77,9 +95,14 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 
 	render() {
 		const iconView = !this.nosort ?
-			html`<d2l-icon icon="${this.desc ? 'tier1:arrow-toggle-down' : 'tier1:arrow-toggle-up'}"></d2l-icon>` :
+			html`<d2l-icon class="d2l-sortable-button-icon" icon="${this.desc ? 'tier1:arrow-toggle-down' : 'tier1:arrow-toggle-up'}"></d2l-icon>` :
 			null;
-		return html`<button type="button"><slot></slot>${iconView}</button>`;
+
+		return html`
+			<div class="d2l-sortable-button-container">
```

Comment:
Is this container actually needed or would we be able to use `:host` in the css?

Reply:
Did you look into why that would be?